packages/strapi-hook-bookshelf/lib/buildQuery.js

```diff
@@ -245,6 +246,10 @@ const buildWhereClause = ({ qb, field, operator, value }) => {
       return qb.where(field, 'like', `%${value}%`);
     case 'ncontainss':
       return qb.whereNot(field, 'like', `%${value}%`);
+    case 'null': {
+      const flag = castValueToType({ type: 'boolean', value: value });
```

Comment:
rather than casting in each buildQuery can you please do this cast directly in the strapi-utils buildQuery function where there is already on cast done.